### ISSUE TITLE
Generate PHPSTORM META doesn't include Mage::getResourceHelper()

### ISSUE THREAD

**Issue by Vinai** [enriquepiatti/Magicento#48]
I really like the option to generate the meta type hint file, thanks for that!
Could the Mage::getResourceHelper() factory method be included, too?

Thanks!

**Comment by enriquepiatti** [comment]
yes sure, anyway in the last code (not released yet because adding the new PHP api is breaking some old features and I need to figure out how to maintain the full compatibility :( ) the meta type hint file generator is not needed anymore, everything is automatic (and it should be faster).
I don't know if it's good to remove or leave that action (create meta file) anyway (it's not needed but maybe is useful for someone, I don't know, for using it with Magicento disabled or in projects where Magicento is not working properly, like those with external libraries), what do you think?

**Comment by Vinai** [comment]
If it doesn't add any additional benefit for working in an installation with Magicento, I'd say remove the option again.

Otherwise it's just one more thing to maintain.
If someone wants to generate the meta file, there are plenty of scripts around that let him do that without Magicento, too.

By the way, I think it's awesome that it will be working automatically in future!
